src/core/sys/posix/aio.d

```diff
@@ -0,0 +1,66 @@
+/**
+ * D header file to interface with the Linux aio API (http://man7.org/linux/man-pages/man7/aio.7.html).
+ * Available since Linux 2.6
+ *
+ * Copyright: Copyright Alexandru Razvan Caciulescu 2018.
```

Comment:
D Language Foundation.

Reply:
I took `core.sys.linux.epoll` as an example for the copyright, is that file not respecting the convention?

Reply:
That file was written long before the DFL existed. We can't easily change the copyright for existing modules, but we can for new additions.

Reply:
fixed